pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/BaseTransformFunctionTest.java

```diff
@@ -89,7 +91,7 @@ public void setUp()
       _floatSVValues[i] = _intSVValues[i] * RANDOM.nextFloat();
       _doubleSVValues[i] = _intSVValues[i] * RANDOM.nextDouble();
       _stringSVValues[i] = Double.toString(_intSVValues[i] * RANDOM.nextDouble());
-
+      _stringAlphaNumericSVValues[i] = "CaMeL";
```

Comment:
You can use `RandomStringUtils.randomAlphabetic(int count)` to randomly generate the test case.